composer.json

```diff
@@ -24,6 +24,10 @@
         "symfony/serializer": "~2.2|~3.0|^4.0",
         "twig/twig": "~1.23|~2.0"
     },
+    "require-dev": {
+        "symfony/phpunit-bridge": "^2.7 || ^3.0",
+        "phpunit/phpunit": "~4.8 || ~5.0"
```

Comment:
if you require the PHPUnit bridge, you could just omit the dependency on PHUnit and use the `simple-phpunit` script shipped with the bridge instead to execute tests